### ISSUE TITLE
correct exercise number for chapter 28-30

### ISSUE THREAD

**Issue by dongzhuoer** [jrnold/r4ds-exercise-solutions#84]
In some chapters, like https://jrnold.github.io/r4ds-exercise-solutions/functions.html, you use "Exercise 19.2.1", in other chapters, like https://jrnold.github.io/r4ds-exercise-solutions/r-markdown.html, you use "Exercise 27.1.1.1".

I suppose that chapter 28-30 should be modified (but I might have missed some other chapters)

**Comment by dongzhuoer** [comment]
The problem is that you should change

```
---
output: html_document
editor_options:
  chunk_output_type: console
---

# R Markdown

## R Markdown Basics

### Exercises

#### Exercise <span class="exercise-number">27.1.1.1</span> {.unnumbered .exercise}
```

to 

```
```
---
output: html_document
editor_options:
  chunk_output_type: console
---

# R Markdown

## R Markdown Basics

### Exercise <span class="exercise-number">27.1.1</span> {.unnumbered .exercise}
```

(using the beginning of https://github.com/jrnold/r4ds-exercise-solutions/blob/master/rmarkdown.Rmd for example)

**Comment by dongzhuoer** [comment]
could you please wait for a while?

**Comment by dongzhuoer** [comment]
I am trying to create a PR

**Comment by dongzhuoer** [comment]
I correct all the chapter names, exercise numbers, etc

**Comment by dongzhuoer** [comment]
closed in #91
